include/roboteam_utils/RobotCommands.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <roboteam_utils/Vector2.h>
+#include <roboteam_utils/Angle.h>
+
+#include <vector>
+
+namespace rtt {
+
+enum class KickType {
+    NORMAL, // Horizontal kicking
```

Comment:
Perhaps use KICK instead of normal? I remember that being used elsewhere and it is clearer than normal

Reply:
Yeah, I was also wondering what I should name it. The reason why I originally went for that was that chipping is also kicking, but just upwards instead of horizontal. Ill rename it to `KICK`